src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        return new Ball(colorSupplier.getRandomColor(),random.nextInt(100));
```

Comment:
```suggestion
        return new Ball(colorSupplier.getRandomColor(),random.nextInt(100));
```
Let's make 100 constant